### DISCUSSION TITLE
how to restore channel.backup from a laptop to hard drive?

### DISCUSSION THREAD

**Discussion by adnateorrounded** [raspiblitz/raspiblitz#4429] [Q&A]
Hi everyone,
I am sorry because I feel the answer is somewhere but I can't find it. Does somebody can explain me how to restore the file from my window laptop?
thanks in advance.

**Comment by rootzoll** [comment]
Not sure what you mean with "restore the file from my window laptop". I guess you have that channel.backup and you have your seed words - with both you can setup a new raspiblitz (use the SSH menus) and you can try to recover your funds.

If you are no longer interested to use a raspiblitz or you had hardware problems you can also try to recover your LND wallet thru the ZAP Windows Desktop app: https://github.com/LN-Zap/zap-desktop/releases

**Comment by adnateorrounded** [comment]
Thanks for answering me.

I had a  ssd crash last year.

I settled up a new configuration with my seeds, raspiblitz image etc.


But my PC had converted my channel  backup file into a BAK file.

I am unable to make it recover.

I should install a new raspiblitz image with a proper channel.backup file
IMO.
But first
How should I convert my file?

Thanks for your concern .

Greetings.

**Reply by rootzoll** [reply]
Try to ranme the file to: channel.backup - you may have to google how to change file extensions under Windows.

**Comment by adnateorrounded** [comment]
Ok, I'll do that.

Thanks a lot.

Le jeu. 7 mars 2024 à 11:46, rootzoll ***@***.***> a écrit :

> Try to ranme the file to: channel.backup - you may have to google how to
> change file extensions under Windows.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/raspiblitz/raspiblitz/discussions/4429#discussioncomment-8705529>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASXZTD3QOZUXSBNFRKLPNYTYXBAO5AVCNFSM6AAAAABDRV3TFGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DOMBVGUZDS>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


--
